src/RedisClusterResourceManager.php

```diff
@@ -89,7 +85,7 @@ private function createRedisResourceFromName(
         float $fallbackTimeout,
         float $fallbackReadTimeout,
         bool $persistent,
-        string $fallbackPassword,
+        array|string|null $fallbackAuthentication,
```

Comment:
can you please provide more detailed `array` type for this method?
I guess the array is either `array{0:non-empty-string}` or `array{0:non-empty-string,1:non-empty-string}`.
I'd be fine having this added as union type or you can use `array{0: non-empty-string, 1?: non-empty-string}`, but we should provide more detailed array information here.